test/test-udp-send-and-recv.c

```diff
@@ -94,8 +95,10 @@ static void cl_send_cb(uv_udp_send_t* req, int status) {
   ASSERT(status == 0);
   CHECK_HANDLE(req->handle);
 
-  r = uv_udp_recv_start(req->handle, alloc_cb, cl_recv_cb);
-  ASSERT(r == 0);
+  if (cl_send_cb_called == 0) {
+    r = uv_udp_recv_start(req->handle, alloc_cb, cl_recv_cb);
+    ASSERT(r == 0);
+  }
```

Comment:
```suggestion
  if (cl_send_cb_called == 0)
    ASSERT_EQ(0, uv_udp_recv_start(req->handle, alloc_cb, cl_recv_cb));
```
Ditto lines 151-154.